_source/_posts/2023-09-14-keycloak-kubernetes-prod.md

```diff
@@ -15,7 +15,17 @@ type: awareness
 ---
 Keycloak is an open-source identity and access management solution that allows you to secure your applications and services with ease. JHipster, on the other hand, is a powerful development platform that provides developers with the tools they need to create modern, scalable, and robust web applications using Spring Boot.
 
-Keycloak is the default OpenID Connect server configured with JHipster. Using Keycloak with JHipster is an excellent way to ensure that your web applications are secure and well-protected from unauthorized access. By integrating Keycloak to your applications, you can easily implement authentication and authorization, manage user identities, and provide secure access to resources.
+Keycloak is the default OAuth 2.0 and OpenID Connect (OIDC) server configured with JHipster. The reason for this is that Keycloak can be run in a local Docker container, and pre-populated with users and client registrations. Of course, you can also [use JHipster with Auth0](https://www.jhipster.tech/security/#auth0)!  
```

Comment:
```suggestion
Keycloak is the default OAuth 2.0 and OpenID Connect (OIDC) server configured with JHipster. This is because Keycloak can be run in a local Docker container, and pre-populated with users and client registrations. Of course, you can [switch to Auth0](https://www.jhipster.tech/security/#auth0), but it requires a bit of configuration.
```